zer0-os/zOS
# feat: web3 login routing

### What does this do?

- Created a `LoginPage` component for the login page
- Made `Web3Login` and `EmailLogin` components which "plug into" the main `Login` page component
- Switch between `Web3Login` and `EmailLogin` based on selected option in `login` saga
- Some refactoring to styles as I moved

### Why are we making this change?

To finish off #627 

### How do I test this?

1. Open the login page
2. Click `Web3 Wallet`
3. Check that wallet selector renders
4. Click `Email`
5. Check that email inputs render

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?
